src/business_logic/transaction/objects/internal_deploy.rs

```diff
@@ -77,6 +77,7 @@ impl InternalDeploy {
         self.contract_hash
     }
 
+    // this function is the same than apply_specific_concurrent_changes
```

Comment:
I think this is redundant

Reply:
i put it in case we needed to check the function on python, as their names are different now, it can be confusing but i will delete it.